### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by ChauPham90**
hi,
I wrote auth route and did the testing, but i did not sovle res.send yet. Do you have any idea? Can you have a look in my code? thanks

### CODE REVIEW

File: vidly-chau/routes/auth.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const Joi = require("joi");
+const _ = require("lodash");
```
Comment (BigBeoU) [inline vidly-chau/routes/auth.js:2]: I think u did not use lodash package in this file

File: vidly-chau/routes/auth.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  if (error) {
+    return res.status("400").send(error.details[0].message);
+  }
+  let user = await Users.findOne({ email: req.body.email });
+  if (!user) return res.status("400").send("invalid email or password");
+
+  const validPassword = await bcrypt.compare(req.body.password, user.password); //return true /false
+  if (!validPassword)
+    return res.status("400").send("invalid email or password");
+  res.send(true);
```
Comment (BigBeoU) [inline vidly-chau/routes/auth.js:20]: in my opinion, we can use JSON web tokens instead of a simple boolean

File: vidly-chau/routes/auth.js (hunk 1 above)
Comment (ChauPham90) [reply vidly-chau/routes/auth.js:2]: will fix it

File: vidly-chau/routes/auth.js (hunk 2 above)
Comment (ChauPham90) [reply vidly-chau/routes/auth.js:20]: ok

**Pull request closed without merging**